### ISSUE TITLE
the dataset format

### ISSUE THREAD

**Issue by YazAbbasMosleh** [voicepowered-ai/VibeVoice-finetuning#26]
Hello bothers!
I hope you all are doing well!

I have fine-tuned a Persian TTS but i cannot inference.
can you provide an inference script for this.
i have fine-tuned using the command:
"""
(vibevoice) ai@mehra:/media/ai/vision/yazdan/tts/VibeVoice-finetuning$ python src/finetune_vibevoice_lora.py     --model_name_or_path models/vibevoice-1.5b     --processor_name_or_path src/vibevoice/processor     --train_jsonl data/train.jsonl     --validation_jsonl data/val.jsonl     --text_column_name text     --audio_column_name audio     --output_dir outputs/persian-1.5b-lora     --per_device_train_batch_size 1     --gradient_accumulation_steps 1     --learning_rate 3e-5     --num_train_epochs 6     --logging_steps 20     --save_steps 200     --eval_steps 200     --report_to wandb     --remove_unused_columns False     --bf16 True     --do_train     --do_eval     --gradient_clipping     --max_grad_norm 0.8     --ddpm_batch_mul 4     --diffusion_loss_weight 1.4     --ce_loss_weight 0.04     --voice_prompt_drop_rate 0.2     --lora_target_modules q_proj,k_proj,v_proj,o_proj,gate_proj,up_proj,down_proj     --lora_r 16     --lora_alpha 64     --train_diffusion_head True     --lr_scheduler_type cosine     --warmup_ratio 0.03     --seed 42
"""

I would appreciate you if guide me to resolve this problem

**Comment by jpgallegoar** [comment]
This is a nice UI
https://github.com/fakerybakery/openvoicelab

**Comment by YazAbbasMosleh** [comment]
thank you very much bro!
